src/components/ErrorComponents/DefaultErrorComponent.tsx

```diff
@@ -53,6 +53,7 @@ const DefaultErrorComponent = (props: DefaultErrorComponentProps) => {
       });
     }
   }, []);
+  const exceptionMessage = (props.error as Error)?.message ? (props.error as Error).message : 'Unhandled UI runtime error';
```

Comment:
Nit: the message should be defined before the `useEffect`.

Reply:
Originally I had it above, but for some reason, when testing it with the message being defined above the useEffect, the error results in the default 'Unhandled' default message 100% of the time. 
I just ran this test again to be sure : https://red-hat-it.sentry.io/issues/4027309030/?project=1484024&query=is%3Aunresolved+%21app_name%3Asubscriptions+%21app_name%3Aimage-builder&referrer=issue-stream

Reply:
Because the error prop can be populated on a second render and the effect is triggered only on a mount. That changes with this PR: https://github.com/RedHatInsights/insights-chrome/pull/2332

Reply:
Yes, I am aware of that `undefined` issue. Can we skip the sentry call if the error is empty? Or do you want to merge the PR as it is?

Reply:
Sorry for the late update. I was on PTO. I made the change so it doesnt sent the empty error message. Good catch. @Hyperkid123